### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gunreet03**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ await Collect(
 
 **Notice:** when chaining methods like `map` or `filter`, you'll receive a collection instance in return. You must actively end the call chain using the `.all()` method to process the collection pipeline and retrieve the final result.
 
-You can directly await the result for methods returning a definite value:
+You can directly await the result for methods returning a definite value. The function returns a new instance of the collection without altering the original input array.
```
Comment (marcuspoehls) [inline README.md:70]: I'd love to keep the colon at the end :)
Can you please add it again?

```suggestion
You can directly await the result for methods returning a definite value. The function returns a new instance of the collection without altering the original input array:
```

**Review by marcuspoehls — CHANGES_REQUESTED** [review]
@gunreet03 Hey Gunreet, thank you for the PR! It's a good addition.

Can you please have a look at this minor refinement?

**Comment by gunreet03** [comment]
Sure I'll. Thank you for taking out time to review.

**Comment by marcuspoehls** [comment]
@gunreet03 Hey Gunreet, any chance to refine this PR?

**Pull request closed without merging**